devel/jreleaser/Portfile

```diff
@@ -49,7 +49,7 @@ destroot {
     copy {*}[glob -dir ${worksrcpath} *] ${target}
 
     # Remove extraneous files
-    delete {*}[glob -directory ${target}/bin *.bat]
+    delete {*}[glob -directory ${target}/bin *..bat]
```

Comment:
It looks like this is causing the CI builds to fail, and will likely fail everywhere.

Did you intend to add a second period...?

Reply:
I thought I had this one fixed already. 😰

Reply:
Fixed & squashed. Sorry about that.

Reply:
No worries, it's easy enough to overlook!